### ISSUE TITLE
[BUG] process definition does not show on UI (工作流在页面不显示问题)

### ISSUE THREAD

**Issue by QQhuxuhui** [apache/incubator-dolphinscheduler#1761]
- Problem Description
After upgrading from 1.1.0 to 1.2.0, individual project workflows cannot be queried on the page, but tasks are still executed normally.

- I solved the problem this way
I found in the t_ds_project table that there is a project with the same name in the exception project, and both projects have the same user_id, but the flag of one project is 0 and the other is 1. This should be the flag used to delete in the previous version. But both items of version 1.2.0 are displayed on the page. Later, I deleted the items that had been deleted before, and then returned to normal.

----

### 问题描述
从1.1.0升级到1.2.0后，个别项目工作流在页面查询不到，但是任务依然正常执行。

### 解决过程
我在t_ds_project发现异常项目存在同名项目，并且这两个项目都有同一个user_id，但是一个项目的flag是0，另一个是1，这应该是之前版本的用来删除的标志。但是1.2.0版本这两个项目都在页面上显示了。后来我又删除了一遍之前已经删除过的项目，然后就恢复正常了。

**Comment by qiaozhanwei** [comment]
please use english to describle your idea

**Comment by QQhuxuhui** [comment]
- Problem Description
After upgrading from 1.1.0 to 1.2.0, individual project workflows cannot be queried on the page, but tasks are still executed normally.

- I solved the problem this way
I found in the t_ds_project table that there is a project with the same name in the exception project, and both projects have the same user_id, but the flag of one project is 0 and the other is 1. This should be the flag used to delete in the previous version. But both items of version 1.2.0 are displayed on the page. Later, I deleted the items that had been deleted before, and then returned to normal.
